airflow/www/views.py

```diff
@@ -1417,7 +1417,9 @@ def rendered_templates(self, session):
 
         logger.info("Retrieving rendered templates.")
         dag: DAG = get_airflow_app().dag_bag.get_dag(dag_id)
-        dag_run = dag.get_dagrun(logical_date=dttm)
+        dag_run = dag.get_dagrun(
+            select(DagRun.run_id).where(DagRun.logical_date == dttm).order_by(DagRun.id.desc()).limit(1)
```

Comment:
```suggestion
            run_id=select(DagRun.run_id).where(DagRun.logical_date == dttm).order_by(DagRun.id.desc()).limit(1)
```

Reply:
do we need to pass session here?

Reply:
We don’t _need_ to, but should. Similar to the `dag.get_dagrun` call below.

Reply:
yep, this one is the only place, we did not use `run_id=`

Reply:
Added it